packages/app/src/app/components/Preview/DevTools/Tests/TestSummary/elements.ts

```diff
@@ -49,13 +49,9 @@ export const TestData = styled.div`
 `;
 
 export const SyncIconStyled = styled(SyncIcon)<{ watching: boolean }>`
-  opacity: 0.7;
-  color: ${props => props.theme['button.hoverBackground']};
-  ${props =>
-    props.watching &&
-    css`
-      opacity: 1;
-    `}
+  && {
```

Comment:
Whoah I haven't seen this before, what does `&&` do?

Reply:
This is new for me too. It's a hacky way to override higher specificity. See [the docs](https://www.styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity) and [this issue](https://github.com/styled-components/styled-components/issues/1253#issuecomment-337871693). You can use how many `&` you want. 🤣